### PR TITLE
feat: Add test mode to ad configs for easier testing

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Passes advertising testmode from the native app to the config. This allows us to put a custom debug field into the app for easier ad testing

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/2b3aa19486dd738101c220929aa8a411c1fa190c/index.html

File: android-app/pages/article.js [hunk 1]
```diff
@@ -56,6 +61,7 @@ const ArticleView = ({ articleId, omitErrors, scale, sectionName }) => {
 };
 
 ArticleView.propTypes = {
+  adTestMode: PropTypes.string.isRequired,
```
[the remark this answers is not in the document]
Comment (tuncaulubilge) [reply android-app/pages/article.js]: We will have this inside the production app as ad team want to change it to test ads in prod. That said, it does make sense it to keep it optional, with default set to empty string

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/eeaeca8d95c8c835e1179f154f36dffe0d1f7632/index.html

File: android-app/pages/article.js [hunk 2]
```diff
@@ -33,8 +32,14 @@ const platformAdConfig = {
...
-const ArticleView = ({ articleId, omitErrors, scale, sectionName }) => {
-  const adConfig = { ...platformAdConfig, sectionName };
+const ArticleView = ({
+  adTestMode,
+  articleId,
+  omitErrors,
+  scale,
+  sectionName
+}) => {
+  const adConfig = { ...platformAdConfig, sectionName, testMode: adTestMode };
```
Comment (colincclark) [inline android-app/pages/article.js]: maybe add a test for this prop?

Comment (tuncaulubilge) [reply android-app/pages/article.js]: I'm planning to write some integration tests in the app, which launches the view and checks for analytics events received. 

For this bit particularly, what do we want tested?

**Pull request merged**